DEVELOPERS_GUIDE.md

```diff
@@ -1,51 +1,59 @@
-# Developer's Guide
+# KaiZen OpenAPI Editor - Developer's Guide
 
-# Building SwagEdit from sources
-Development of SwagEdit should be done with Eclipse. This project uses Maven/Tycho so you should have the Eclipse maven plugin `m2e` already installed.
+# Building from Sources
+Development of KaiZen Editor should be done with Eclipse. This project uses Maven/Tycho so you should have the Eclipse maven plugin `m2e` already installed.
 
 ## 1. Get sources
-To start developing SwagEdit, clone the repository with the following command:
+To start developing, clone the repository with the following command:
 
 ```
-git clone git@github.com:ModelSolv/SwagEdit.git
+git clone git@github.com:RepreZen/KaiZen-OpenAPI-Editor.git
 ``` 
 
 ## 2. Import to Eclipse IDE
-Open Eclipse and select `File > Import... > Maven > Existing Maven Project` and select the folder SwagEdit.
-This will put the project SwagEdit into your current workspace. 
+Open Eclipse and select `File > Import... > Maven > Existing Maven Project`. Then select the folder KaiZen-OpenAPI-Editor, 
+and add the following projects into your current workspace:
 
-## 3. Launch: From development Eclipse as Eclipse Application
-From your Eclipse IDE with SwagEdit sources, open the "Launch configurations..." dialog, then right-click on the "Eclipse Application" located in the left side and choose "New". 
+<img width="663" alt="screen shot 2017-05-09 at 11 28 15 am" src="https://cloud.githubusercontent.com/assets/644582/25858815/b3248a80-34aa-11e7-826b-e6b5547a01ed.png">
 
-Make sure that SwagEdit plugins are included to the configuration, for example, you can select the "All workspace and enabled target plag-ins" in the "Plug-ins" tab:
+## 3. Set the Target Platform
 
-## 3. Launch: using a local update site built from sources
+The target platform provides some third-party dependencies, e.g. JSON Schema validator.
 
-From inside your SwagEdit folder, run the following command:
+TO BE COMPLETED...
+
+## 4. Launch: From development Eclipse as Eclipse Application
```

Comment:
Yes, @tedepstein they are alternatives `a` and `b`, one OR another.  
* **Running from sources** is faster and easier. It also provides debugging capabilities.
* **Using a local update site** allows us to test exactly what the users will get, as oftentimes Eclipse ignores `build.properties`. For example, icons can be displayed correctly when launched from Eclipse, but shown as red rectangles in the product. It happens if the containing folder was not added to the build.properties. Another advantage of using a locally built update site is that you can easily share it with other people.